package.json

```diff
@@ -34,20 +34,20 @@
   },
   "dependencies": {
     "@homebound/form-state": "^2.1.5",
-    "@react-aria/utils": "^3.8.1",
+    "@react-aria/utils": "^3.9.0",
```

Comment:
I always wondered why this wasn't part of the general package `react-aria/utils`